### ISSUE TITLE
Atlas-iOS build via Carthage is skipped

### ISSUE THREAD

**Issue by cwalo** [layerhq/Atlas-iOS#1147]
When running `carthage update`:
`*** Skipped building Atlas-iOS due to the error:
Dependency "Atlas-iOS" has no shared framework schemes`

Carthage version: 0.13
Cartfile:
```
github "ReactiveCocoa/ReactiveCocoa" "swift2"
github "Alamofire/Alamofire" >= 3.0
github "puffinsupply/Restraint" >= 1.0
github "tristanhimmelman/AlamofireObjectMapper" ~> 2.0
github "layerhq/Atlas-iOS"
```

**Comment by cwalo** [comment]
I did a little more digging and noticed that the Atlas framework scheme was not shared. After sharing the framework scheme, build actually starts but throws an error outlined in issue [#1149](https://github.com/layerhq/Atlas-iOS/issues/1149).

**Comment by tomco** [comment]
Have a look at #1139. I created a pull request to fix the issue. Unfortunately it has not been merged yet.

**Comment by feifanzhou** [comment]
Take a look at https://github.com/layerhq/Atlas-iOS/pull/1144/files — included changes in #1139 for Carthage compatibility

**Comment by feifanzhou** [comment]
Resolved by #1181
